aquacomputer_d5next.c

```diff
@@ -1958,6 +2066,8 @@ static int aqc_probe(struct hid_device *hdev, const struct hid_device_id *id)
 		priv->secondary_ctrl_report_id = AQUASTREAMXT_SECONDARY_CTRL_REPORT_ID;
 		priv->secondary_ctrl_report_size = AQUASTREAMXT_SECONDARY_CTRL_REPORT_SIZE;
 		priv->secondary_ctrl_report = aquastreamxt_secondary_ctrl_report;
+	} else if (priv->kind == aquastreamult) {
+		priv->fan_structure = &aqc_aquastreamult_fan_structure;
 	} else {
 		priv->serial_number_start_offset = AQC_SERIAL_START;
 		priv->firmware_version_offset = AQC_FIRMWARE_VERSION;
```

Comment:
I assume this won't work as the serial number and firmware version offsets are never set for the Aquastream Ultimate.